packages/edit-site/src/components/sidebar-dataviews/default-views.js

```diff
@@ -58,10 +64,10 @@ const DEFAULT_POST_BASE = {
 	page: 1,
 	perPage: 20,
 	sort: {
-		field: 'date',
-		direction: 'desc',
+		field: 'title',
```

Comment:
Why change the default sort here?

Reply:
It was suggested [here](https://github.com/WordPress/gutenberg/pull/66479#issuecomment-2494024628). Note that wp-admin uses a mix of `menu_order` and `title` (see [code](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/includes/class-wp-posts-list-table.php#L169)).